lib/hawkular/base_client.rb

```diff
@@ -175,6 +178,16 @@ def tenant_header
       headers
     end
 
+    # @!visibility private
```

Comment:
imo there's no much point to have it here but I am not strongly opinionated regarding this